schemas/common_definitions.json

```diff
@@ -271,6 +271,11 @@
               "type": "string",
               "description": "The id of another group. The first block in that group will be routed to."
             },
+            "section": {
+              "type": "string",
+              "description": "Routes to the end of a section",
+              "enum": ["end"]
```

Comment:
Our enums are capitalised to differentiate them from other values, this should be `End`.